code/game/gamemodes/blob/blobs/shield.dm

```diff
@@ -5,6 +5,7 @@
 	desc = "A solid wall of slightly twitching tendrils."
 	health = 150
 	maxhealth = 150
+	brute_resist = 0.1

```

Comment:
this is like 1500 health versus brute don't even bother with a wrench unless it's electromagnetic web